tools/windows/prepare-binary-addons-dev.bat

```diff
@@ -76,7 +76,7 @@ IF "%addon%" NEQ "" (
 )
 
 rem execute cmake to generate Visual Studio 12 project files
```

Comment:
i dont want to get into how windows uses this, but maybe update this comment from "Visual Studio 12" just to be inline with the actual cmake generator.

My query would be, should the generator be hard coded, or should an env var or whatever be used, with a potential fall back to a "default" generator version if not set?

Reply:
This is just a development script (note the path, production scripts use `tools/buildsteps/windows`) so hard-coding is fine.